test/modules/TestAnalyzer/Settings.cs

```diff
@@ -41,7 +40,12 @@ class Settings
             this.WebhostPort = Preconditions.CheckNonWhiteSpace(webhostPort, nameof(webhostPort));
             this.ToleranceInMilliseconds = Preconditions.CheckRange(tolerance, 0);
             this.StoragePath = storagePath;
-            this.TestInfo = testInfo.Split(",", StringSplitOptions.RemoveEmptyEntries);
+            this.TestInfo = testInfo.Split(",", StringSplitOptions.RemoveEmptyEntries)
```

Comment:
Beware that StringSplitOptions might not work as you expect.
https://stackoverflow.com/questions/10682301/stringsplitoptions-removeemptyentries-doesnt-work-as-advertised/10682341

```"      " by definition is not empty (it is actually whitespace), so it is not removed from resulting array.```

Reply:
it should be fine since I ensure '=' should be there.